core/src/main/java/io/grpc/internal/ClientCallImpl.java

```diff
@@ -347,10 +349,12 @@ private void removeContextListenerAndCancelDeadlineFuture() {
 
     @Override
     public void run() {
+      ToStringHelper timeoutDetails = MoreObjects.toStringHelper("");
```

Comment:
a comment saying why this is blank would be helpful.  It's unusual, and not immediately obvious.